knockout-pre-rendered.js

```diff
@@ -121,20 +121,93 @@
     };
   }
 
-  function InitializedForeach(spec) {
+  // Knockout automatically generates "_ko_property_writers" for a subset of its
+  // built-in bindings, registered as "_twoWayBindings". Here, we can compel it to
+  // do the same for the rest of the bindings that we want to support with "init".
+  ko.utils.extend(ko.expressionRewriting._twoWayBindings, {
```

Comment:
How safe it is to use this `_twoWayBindings` property? To me, the underscore seems to imply that it might be a private field that is not intended to be used by third parties. Is my view incorrect?

Reply:
From [src/binding/expressionRewriting.js](https://github.com/knockout/knockout/blob/master/src/binding/expressionRewriting.js#L199-L201):
```
// For those developers who rely on _ko_property_writers in their custom bindings, we expose _twoWayBindings as an
// undocumented feature that makes it relatively easy to upgrade to KO 3.0. However, this is still not an official
// public API, and we reserve the right to remove it at any time if we create a real public property writers API.
```
They expose it as an undocumented feature, for people making their own bindings. So although it's not guaranteed as future-proof, I don't see it as totally off limits. The only side-effect this has on knockout's own internal bindings is to generate _ko_property_writers for them.

Reply:
If this troubles you, I can rework it to generate pre-processors for those bindings, which will generate custom `_ko_prerendered_initPropertyWriters`, the same way that is being done for the `init` and `attr` bindings.  But this seems redundant to what KO's own preprocessor already does for these bindings, so I figured it was simpler to leverage that.

Reply:
Aha, that explanation makes it look like it is fine to use `_twoWayBindings`.